client/question/question.js

```diff
@@ -723,6 +724,14 @@ tie.constant('SECONDS_TO_MILLISECONDS', 1000);
  */
 tie.constant('CODE_CHANGE_DEBOUNCE_SECONDS', 5);
 
+/** Default user stdOut separator length.
```

Comment:
Perhaps give a bit more context about what the "stdout separator" is and how/why it's used.

Reply:
Done.